scripts/build.ps1

```diff
@@ -35,7 +35,10 @@ function Generate() {
   if ($ci) {
   conan config install https://github.com/novelrt/ConanConfig.git
   conan install . -if "$BuildDir" --build=missing --profile windows-vs2019-amd64
```

Comment:
Shouldn't this be `..`? I know its not part of this PR but its something I just spotted.

Reply:
Nope, since this is working in the root directory, it's pointing accurately to `conanfile.py` 😄

Reply:
That's wild. How is that even building then?

Reply:
We pipe in the appropriate build/repo paths when calling the respective functions (i.e `cmake -S $RepoRoot -B $BuildDir` to configure, `cmake --build $BuildDir` to build, etc) so that everything is where it should be, even though we're calling from the root directory